requirements/pytorch/extra.txt

```diff
@@ -5,7 +5,7 @@
 matplotlib>3.1, <3.9.0
 omegaconf >=2.0.5, <2.4.0
 hydra-core >=1.0.5, <1.4.0
-jsonargparse[signatures] >=4.26.1, <4.28.0
+jsonargparse[signatures] >=4.27.5, <4.28.0
```

Comment:
Do existing users who do not enable this feature need to upgrade too? If the answer is yes, then you should update the `_JSONARGPARSE_SIGNATURES_AVAILABLE` global in cli.py

Reply:
Done